unittests/integration/ghz_nisq_tester.cpp

```diff
@@ -94,6 +94,8 @@ CUDAQ_TEST(GHZSampleTester, checkBroadcast) {
   }
 }
 
+#ifndef CUDAQ_BACKEND_TENSORNET
+// Tensornet backend doesn't support seed.
```

Comment:
Is there a plan to support this in the future?

Reply:
Unresolving this, since I have the same question - why not support this now?

Reply:
It is a limitation of the `cutensornetsampler` as documented [here](https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/api/functions.html#cutensornetsamplersample) (will be addressed in the next release).

Reply:
Ok. Could you please file a GitHub issue for this and link it here?

Reply:
I've created this [issue](https://github.com/NVIDIA/cuda-quantum/issues/895) for tracking purposes.